### PR TITLE
Pillbox/CamPillbox blind spot fix

### LINKED ISSUE #9529: Pillbox does not shoot infantry on a specific spot
force fire gets rejected ... and it does not get triggered when its a enemy unit 

pillbox fire

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/27145347-pillbox-does-not-shoot-infantry-on-a-specific-spot?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by HenrytheSlav**
Fixes #9529. It is just lame hack, but it works. If you feel like this could break something then tell me and I'll throw it away/change values. Increasing by 1 was not enough so I arbitrarily chose 100.

### CODE REVIEW

File: mods/ra/rules/structures.yaml [hunk 1]
```diff
@@ -568,7 +568,7 @@ PBOX:
 		Armaments: garrisoned
 		PortOffsets: 384,0,128, 224,-341,128, -224,-341,128, -384,0,128, -224,341,128, 224,341,128
 		PortYaws: 0, 176, 341, 512, 682, 853
-		PortCones: 86, 86, 86, 86, 86, 86
+		PortCones: 100, 100, 100, 100, 100, 100
```
Comment (pchote) [inline mods/ra/rules/structures.yaml]: IIRC from my own earlier tests increasing these to 87 should be sufficient and will avoid graphical glitches (firing out the wrong hole).

Comment (GraionDilach) [inline mods/ra/rules/structures.yaml]: I'd prefer 88 though, 87 might have rounding issues.

Comment (HenrytheSlav) [inline mods/ra/rules/structures.yaml]: As I said, 87 didn't work.Might try use lowest working. I have map with tons of unmoving engineers in every cell around pbox to test with.

**Comment by HenrytheSlav** [comment]
Updated to 88, that works and should cause no issues.

**Comment by GraionDilach** [comment]
:+1:

**Comment by abcdefg30** [comment]
Looks good to me. :+1: / :white_check_mark:

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/b7d86f4cf5ca8acc0876677b193d8fb66488d2e1...e9cea306e1e0e3ef579975143057ce4f8a0588d3)

**Pull request merged**